[Governance] Add secret example to custom build policy examples (#663)
hlxsites/prisma-cloud-docs: docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policy-examples.adoc

----

[#ss-examples]
=== Secrets Security Examples

> The following examples demonstrate how to create a custom build policy code.

Reviewer: ```suggestion
The following examples demonstrate how to create a custom build policy for detecting secrets.
```

//* <<ss-basic-policy-definition, Basic Policy Definition>>

[#ss-basic-policy-definition]
==== Example #1 Basic Policy Definition

> Use the sample guideline to create a basic policy definition using a single attribute.

Reviewer: ```suggestion
Use the sample guideline to create a basic policy definition using a single attribute. Multiple signatures, as shown in this example, will be treated as OR statements, meaning that all strings matching either pattern will be flagged.
```

    - "[A-Za-z0-9]{8,20}"
    - "my-super-secret-password-regex"
----

> ==== Example #2 Detect Passwords

Reviewer: ```suggestion
==== Example #2 Detect Generic Passwords
```

> This policy detects instances where literal strings starting with "password" or "pwd" are used directly in the code or configuration files. This is a security risk because it exposes the actual password value.

Reviewer: ```suggestion
This policy detects instances where literal strings starting with "password", "pass" or "pwd" are used directly in the code or configuration files. This is a security risk because it exposes the actual password value.
```

----
metadata:

>   name: "Detect password definition"

Reviewer: ```suggestion
```

Reviewer: Just guidelines and category are needed in the platform

>   guidelines: "Find passwords starting with password|pwd and the value format is not {var}.{var}"

Reviewer: ```suggestion
  guidelines: "Find passwords starting with password|pass|pwd and the value format is not {var}.{var}"
```

>   category: "secrets"
>   severity: "high"

Reviewer: ```suggestion
```
